zebra/if_netlink.c

```diff
@@ -432,7 +432,7 @@ static uint32_t get_iflink_speed(struct interface *interface, int *error)
 			/* no vrf socket creation may probably mean vrf issue */
 			if (error)
 				*error = INTERFACE_SPEED_ERROR_READ;
-			return 0;
+			return DEFAULT_SPEED;
```

Comment:
Why is setting the speed to `DEFAULT_SPEED` acceptable when we don't know the speed of the link?  How is this better?

Reply:
you re right. this commit is removed.